utils/numbers.go

```diff
@@ -8,11 +8,12 @@ import (
 	"time"
 )
 
-func NewUint64(val uint64) *uint64 { return &val }
+// PointerTo returns a pointer to the given value.
+func PointerTo[T any](x T) *T { return &x }
```

Comment:
The function should be moved here, rather than `utilstest` as it not only used in tests.